pkg/services/control/server/status.go

```diff
@@ -0,0 +1,95 @@
+package control
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/nspcc-dev/neofs-node/pkg/services/control"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (s *Server) ObjectStatus(_ context.Context, request *control.ObjectStatusRequest) (*control.ObjectStatusResponse, error) {
+	err := s.isValidRequest(request)
+	if err != nil {
+		return nil, status.Error(codes.PermissionDenied, err.Error())
+	}
+
+	// check availability
+	err = s.ready()
+	if err != nil {
+		return nil, err
+	}
+
+	var addr oid.Address
+	err = addr.DecodeString(string(request.Body.ObjectAddress))
+	if err != nil {
+		return nil, status.Error(codes.Internal, fmt.Sprintf("parsing object address: %s", err.Error()))
```

Comment:
```suggestion
		return nil, status.Errorf(codes.Internal, "parsing object address: %v", err)
```
same below

Reply:
ok, but still %s if you dont mind

Reply:
`%s` for `error` is `err.Error()`, no need to call the method

Reply:
yes, that is why i changed it to `status.Errorf`, dropped `.Error` call but kept `%s` (not `%v`), that is why my answer

Reply:
oh i see now, u just did it diff in two cases